src/server/models/user.js

```diff
@@ -471,6 +471,13 @@ module.exports = function(crowi) {
     return this.findOne({username});
   };
 
+  userSchema.statics.findUserByEmail = function(email) {
+    if (email == null) {
+      return Promise.resolve(null);
+    }
+    return this.findOne({email});
+  };
```

Comment:
Mongoose have `Mode.findOne(query): Promise<Model>` method.
We think `findUserByHoge` method is no longer needed, We plan to delete `findUserByUsername` and `findUserByApiToken` in the near future. 

Please use `findOne` in `external-account.js` directly.